src/components/Patient/ManageEncounterOrganizations.tsx

```diff
@@ -81,22 +82,21 @@ export default function ManageEncounterOrganizations({
         {trigger || (
           <Button variant="outline" size="sm">
             <Building className="mr-2 h-4 w-4" />
-            Manage Organizations
+            {t("manage_organizations")}
           </Button>
         )}
       </SheetTrigger>
       <SheetContent>
         <SheetHeader>
-          <SheetTitle>Manage Organizations</SheetTitle>
+          <SheetTitle>{t("manage_organizations")}</SheetTitle>
           <SheetDescription>
-            Add or remove organizations from this encounter
+            {t("manage_organizations_description")}
```

Comment:
Check what manage_organizations_description stands for in en.json. This won't work.